packages/map-template/src/components/MapsIndoorsMap/MapsIndoorsMap.jsx

```diff
@@ -110,6 +113,7 @@ function MapsIndoorsMap(props) {
 			category: props.supportsUrlParameters && categoryQueryParameter ? categoryQueryParameter : props.category,
             searchAllVenues: props.supportsUrlParameters && searchAllVenuesParameter ? searchAllVenuesParameter : (props.searchAllVenues || defaultProps.searchAllVenues),
             hideNonMatches: props.supportsUrlParameters && hideNonMatchesQueryParameter ? hideNonMatchesQueryParameter : props.hideNonMatches,
+            showRoadNameLabels: props.supportsUrlParameters && !isNullOrUndefined(showRoadNameLabelsQueryParameterBoolean) && !isNullOrUndefined(queryStringParams.get('showRoadNameLabels')) ? showRoadNameLabelsQueryParameterBoolean : props.showRoadNameLabels
```

Comment:
Do we know if `isNullOrUndefined` should be used on other "boolean" props?

Reply:
I believe not for now. This specific boolean is meant to be undefined in specific cases. Looking at other properties, I do not see a case where undefined is a desired value.